### PR TITLE
Change code structure

### PR DESCRIPTION
**Pull request by mcygoogle**

### CODE REVIEW

File: src/main/java/com/google/googleidentity/oauth2/request/RefreshTokenRequestHandler.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
     limitations under the License.
 */
 
-package com.google.googleidentity.oauth2.refresh;
+package com.google.googleidentity.oauth2.request;
```
Comment (weiminhuang-corp) [inline src/main/java/com/google/googleidentity/oauth2/request/RefreshTokenRequestHandler.java:17]: can this class be final?

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
         <env-var name="DB_USER" value="root" />
         <env-var name="DB_PASS" value="123456789" />
         <env-var name="DB_NAME" value="oauth2demo" />
-        <env-var name="USE_CLOUD_SQL" value="true" />
```
Comment (weiminhuang-corp) [inline src/main/webapp/WEB-INF/appengine-web.xml:12]: Why changed this?

File: src/test/java/com/google/googleidentity/oauth2/endpoint/AuthorizationEndpointTest.java [hunk 3]
```diff
@@ -94,14 +87,9 @@ public void init() {
     userDetailsService.addUser(USER);
     userSession = new UserSession();
     userSession.setUser(USER);
-    Map<GrantType, RequestHandler> map = new HashMap<>();
```
Comment (weiminhuang-corp) [inline src/test/java/com/google/googleidentity/oauth2/endpoint/AuthorizationEndpointTest.java:97]: Can the test cases pass without setup the expected respond from the mock?

File: src/test/java/com/google/googleidentity/oauth2/endpoint/TokenEndpointTest.java [hunk 4]
```diff
@@ -110,13 +103,8 @@ public void init() {
...
     clientSession.setClient(CLIENT);
     System.setProperty("AUTH_CODE_LENGTH", "10");
-    Map<GrantType, RequestHandler> map = new HashMap<>();
-    map.put(
-        GrantType.AUTHORIZATION_CODE,
-        new AuthorizationCodeRequestHandler(
-            new AuthorizationCodeService(new InMemoryCodeStore()),
-            new InMemoryOAuth2TokenService()));
-    tokenEndpoint = new TokenEndpoint(clientDetailsService, new MultipleRequestHandler(map));
+    RequestHandler requestHandler = mock(RequestHandler.class);
```
Comment (weiminhuang-corp) [inline src/test/java/com/google/googleidentity/oauth2/endpoint/TokenEndpointTest.java:106]: Same here, I think you need to set the expected request and response pair in order to pass the test cases.

File: src/test/java/com/google/googleidentity/oauth2/endpoint/AuthorizationEndpointTest.java (hunk 3 above)
Comment (mcygoogle) [reply src/test/java/com/google/googleidentity/oauth2/endpoint/AuthorizationEndpointTest.java:97]: They can, actually I did not test the handlers here.

File: src/test/java/com/google/googleidentity/oauth2/endpoint/TokenEndpointTest.java (hunk 4 above)
Comment (mcygoogle) [reply src/test/java/com/google/googleidentity/oauth2/endpoint/TokenEndpointTest.java:106]: The handler's responses are tested in other tests.

File: src/main/webapp/WEB-INF/appengine-web.xml (hunk 2 above)
Comment (mcygoogle) [reply src/main/webapp/WEB-INF/appengine-web.xml:12]: In memory setting can reduce the time for launching the application. It is better for me to test and modify the UI.

**Pull request merged**